atc/exec/set_pipeline_step_test.go

```diff
@@ -305,6 +309,99 @@ jobs:
 					Expect(succeeded).To(BeTrue())
 				})
 			})
+
+			Context("when team is configured", func() {
+				var (
+					fakeUserCurrentTeam *dbfakes.FakeTeam
+				)
+
+				BeforeEach(func() {
+					fakeUserCurrentTeam = new(dbfakes.FakeTeam)
+					fakeUserCurrentTeam.IDReturns(111)
+					fakeUserCurrentTeam.NameReturns("main")
+					fakeUserCurrentTeam.AdminReturns(false)
+
+					stepMetadata.TeamID = fakeUserCurrentTeam.ID()
+					stepMetadata.TeamName = fakeUserCurrentTeam.Name()
+					fakeTeamFactory.FindTeamReturnsOnCall(
+						0,
+						fakeUserCurrentTeam, true, nil,
+					)
+				})
+
+				Context("when team is empty", func() {
```

Comment:
nit: this Context sounds like the opposite of the outer Context which is a little confusing

Reply:
Force pushed so that the inner context is more specific "when team is set to the empty string"